web-application/webpage-with-ajax-call/style.css

```diff
@@ -0,0 +1,26 @@
+body {
+  font-family: Arial, sans-serif;
+  margin: 0;
+  padding: 0;
+  background-image: url('https://img.freepik.com/free-photo/assortment-cinema-elements-red-background-with-copy-space_23-2148457848.jpg'); 
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+}
+
+#container {

```

Comment:
For styles always use `class` instead of `id` whenever possible.

Reply:
Uses class instead of id.